### DISCUSSION TITLE
Not able to get approved for teacher benefits

### DISCUSSION THREAD

**Discussion by comicsgirl5100** [community/community#57243] [GitHub Education]
### Hi everyone,

<img width="656" alt="Screen Shot 2023-06-05 at 9 33 57 AM" src="https://github.com/community/community/assets/34179669/9b5b07ec-b37d-4c6a-b532-fdc58a1445d7">

I have tried uploading my ID and even a pay stub for proof of being a teacher and it keeps giving me the same responses. Any suggestions? My coworker uploaded his ID and it worked.

**Comment by dakshdeepHERE** [comment]
Hmm I haven't applied for the teacher benefits but have for student's benefit and this same issue happened with me. When my request was getting rejected again and again when I was using my ID card to verify I think the verification are done with bots and they need some specific points on the document like for student pack they confirm name and match it with your github name email address your education time period for my case it is 2021-2025 and other info too they are all mentioned there when you are trying to apply for the benefits. Also what helped me was connecting multiple documents and taking clear picture of my id. Hope this helps you @comicsgirl5100

**Comment by samus-aran** [comment]
Hi @comicsgirl5100 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
